.github/workflows/dev_deploy.yml

```
@@ -0,0 +1,165 @@
+---
+name: Deploy/Destroy Branch Dev Environment
+
+on:
+  pull_request:
+    types: [labeled, closed]
+    
+env:
+  TERRAFORM_CLOUD_TOKENS: app.terraform.io=${{ secrets.HUSHLINE_DEV_TF_TOKEN }}
+  GITHUB_TOKEN: ${{ github.token }}
+  WORKSPACE_NAME: hushline-dev-${{ github.head_ref }}
+  TF_PROJECT_HUSH_LINE_DEV: prj-iEruEQFmaNTCRAtA
+  DEV_TF_PATH: terraform/dev
+  DO_APP_NAME: hushline-dev-${{ github.head_ref }}
+  HUSHLINE_INFRA_REPO: scidsg/hushline-infra
+  HUSHLINE_INFRA_REF: dev-team
```

TODO:
```suggestion
  HUSHLINE_INFRA_REF: main
```
after https://github.com/scidsg/hushline-infra/pull/50 is merged